gwnr/waveform/hybridize.py

```diff
@@ -313,9 +341,23 @@ def hybridize_modes(
     the one at the rightmost occurance in time) 
 
     """
-    t2_index_insp = find_last_value_location_in_series(
-        frq_insp[(el, em)], frq_attach + frq_width / 2
-    )
+    if hybridize_using_avg_orbital_frequency:
+        t2_index_insp = find_last_value_location_in_series(
+            inspiral_orbital_frequency, (frq_attach + frq_width / 2) / em
+        )
```

Comment:
This function (`hybridize_modes()`) assumes `frq_attach` and `freq_width` should be in accordance with the `em` value of `mode_to_align_by`. Hence change this to
```
 t2_index_insp = find_last_value_location_in_series(
            inspiral_orbital_frequency, (frq_attach + frq_width / 2)
        )
```